CHANGELOG.md

```diff
@@ -17,7 +17,7 @@ Older Logs can be found in `docs/RELEASE-NOTES-*`.
 #### gr-analog
 - `fmdet_cf`'s derivative coefficients were wrong.
 #### gr-blocks
-- `float_array_to_int`: int32 limits were wrongly hardcoded. 
+- `float_array_to_int`: int32 limits were wrongly hardcoded.
```

Comment:
sorry about this whitespace fix, it crept in there by accident. I had already committed other commits subsequently and so it would of been messy to revert.